### DISCUSSION TITLE
Verify contract error

### DISCUSSION THREAD

**Discussion by samehsadaqa** [NomicFoundation/hardhat#4772] [Troubleshooting]
Hi team,
I'm trying to verify a contract on mainnet, but I'm getting this error
hardhat-verify found one or more errors during the verification process:

Etherscan:
An unexpected error occurred during the verification process.
Please report this issue to the Hardhat team.
Error Details: getaddrinfo ENOTFOUND api.etherscan.io

when I try to verify the same contract code on testnet it works, any help please

**Comment by fvictorio** [comment]
This is very strange. Are you still running into this issue? And my understanding is that this happens to you on mainnet but not on testnets, is that correct?

**Reply by schaable** [reply]
Hey @samehsadaqa if you are still running into this issue, can you share your etherscan config **(omit any sensitive info like apiKey)**?
